src/pytest_plugins/execute/eth_config/test_test_eth_config.py

```diff
@@ -0,0 +1,141 @@
+"""Unit tests for the `eth_config` execute tests."""
```

Comment:
I'd put this is in a `./tests/` sub-dir to match repo standards and avoid the `test_test` prefix :smile: